MAINTAINERS.md

```diff
@@ -0,0 +1,8 @@
+# Maintainers
+
+The following table lists Topaz project maintainers and areas of expertise in alphabetical order:
+
+| Name | GitHub | Email | Organization | Repositories/Area of Expertise | Added/Renewed On |
+| --- | --- | --- | --- | --- | --- |
+| Gert Drapers | @gertd | gert@aserto.com | Aserto | all repositories | 2022-10-23 |
+| Dani Carabas | @carabasdaniel | dani@aserto.com | Aserto | all repositories | 2022-10-23 |
```

Comment:
The list should be sorted in alphabetical order, Dani sorts on top :)

Reply:
done